MonoGame.Framework/Color.cs

```diff
@@ -174,8 +186,8 @@ private Color(uint packedValue)
 			//_packedValue = (packedValue & 0xff00ff00) | ((packedValue & 0x000000ff) << 16) | ((packedValue & 0x00ff0000) >> 16);
         }
 
-	/// <summary>
-        /// Creates a new instance of <see cref="Color"/> struct.
+        /// <summary>
+        /// Constructs a color.

```

Comment:
How about...
```
Constructs an RGBA color from the XYZW unit length components of a vector.
```
At least then we are telling users the implied order (X is R, Y is G, etc.) and that they are assumed to be 0 to 1.

This same idea can be applied to the other constructors here.